src/dmd/cparse.d

```diff
@@ -1416,13 +1416,31 @@ final class CParser(AST) : Parser!AST
         {
             Identifier id;
             AST.Expression asmname;
-            auto dt = cparseDeclarator(tspec, id);
+            AST.Type dt;
+            if (level == LVL.member && token.value == TOK.colon)
+            {
+                // C11 6.7.2.1-12 unnamed bit-field
+                nextToken();
+                cparseConstantExp();
+                error("unnamed bit fields are not supported"); // TODO
+                dt = AST.Type.tuns32;
```

Comment:
This should really be `dt = tspec`, but actually dealing with bit-fields will be done in a follow-up.

Reply:
It would be nice to put this comment in the code.

Reply:
I'd rather just deal with it upon implementing bit-fields.